rdf_differ/diff_adapter.py

```diff
@@ -0,0 +1,344 @@
+"""
+diff_adapter.py
```

Comment:
Ignore this file. it is shown because it was renamed from `diff_getter` to `diff_adapter`